datadog/api/resources.py

```diff
@@ -35,18 +38,19 @@ def create(cls, attach_host_name=False, method='POST', id=None, params=None, **b
             params = {}
 
         path = cls._resource_name
+        api_version = getattr(cls, '_api_version', _api_version)
```

Comment:
Isn't this redundant here and before all other calls to `submit` in this file since you are also defaulting to `v1` in the implementation of `submit` ? I think it is sufficient to default it to `v1` in `api_client.py` only.